core/default_eventpage_content.py

```diff
@@ -0,0 +1,84 @@
+# -*- encoding: utf-8 -*-
+import random
+
+from django.conf import settings
+from django.core.files import File
+
+
+DEFAULT_BACKGROUND_PHOTOS = {
+    'about': [
+        settings.STATICFILES_DIRS[0]+'/img/photos/about1.jpg',
+        settings.STATICFILES_DIRS[0]+'/img/photos/about2.jpg'
+    ],
+    'apply': [
+        settings.STATICFILES_DIRS[0]+'/img/photos/apply.jpg'
+    ],
+    'coach': [
+        settings.STATICFILES_DIRS[0]+'/img/photos/coach1.jpg',
+        settings.STATICFILES_DIRS[0]+'/img/photos/coach2.jpg'
+    ],
+    'footer': [
+        settings.STATICFILES_DIRS[0]+'/img/photos/footer1.jpg',
+        settings.STATICFILES_DIRS[0]+'/img/photos/footer2.jpg'
+    ]
+}
+
+def get_random_photo(section):
+    if section in DEFAULT_BACKGROUND_PHOTOS:
+        photos = DEFAULT_BACKGROUND_PHOTOS[section]
+        return File(open(photos[random.randint(0, len(photos)-1)], 'rb'))
+    return None
+
+
+def get_default_eventpage_data():
+    return [
+        {
+            'name': 'about',
+            'is_public': True,
+            'background': get_random_photo('about'),
+            'content': '<div style="text-align: center;"><h1>Free programming workshop for women</h1><h2>Build your first website at EuroPython 2014 in Berlin!</h2><a class="btn" href="#value">Learn more »</a></div>'
```

Comment:
I believe putting in a template and loading it here would be much more readable for further changes. Maybe a #TODO if you don't have time for that now? :)